src/common/modules/navigations/gnb/modules/gnb-search/GNBSearch.vue

```diff
@@ -141,23 +120,41 @@ export default {
                 });
             }),
             cloudServiceItems: computed<SuggestionItem[]>(() => {
-                const cloudServiceList = state.showRecent ? state.recentCloudServiceList : state.cloudServiceList;
-                return cloudServiceList.map(d => ({
+                if (state.showRecent) return state.recentCloudServiceList;
+                return state.cloudServiceList.map(d => ({
                     name: d.id,
                     label: d.name,
                     icon: d.icon,
                     defaultIcon: d.icon,
                     parents: [{ name: d.group, label: d.group }],
+                    provider: d.provider,
                 }));
             }),
             isFocusOnInput: false,
             isFocusOnSuggestion: false,
             focusStartPosition: 'START',
             showRecent: computed(() => state.visibleSuggestion && !state.inputText.length),
             recentMenuList: [] as SuggestionItem[],
-            recentCloudServiceList: [] as CloudService[],
+            recentCloudServiceList: [] as SuggestionItem[],
         });
 
+        /* Util */
+        const getConvertedCloudServiceList = (rawData): CloudService[] => {
+            const cloudServiceList: CloudService[] = [];
+            rawData.forEach((d) => {
+                const data = state.cloudServiceTypes[d.key];
+                cloudServiceList.push({
+                    id: d.key,
+                    name: d.data.name,
+                    group: d.data.group,
+                    provider: d.data.provider,
+                    icon: data.icon,
+                });
+            });
+            return cloudServiceList;
+        };
```

Comment:
```suggestion
        const getConvertedCloudServiceList = (rawData): CloudService[] => {
           return rawData.map(d => {
               return {
                   id: d.id,
                   name: d.data.name,
                   group: d.data.group,
                   provider: d.data.provider,
                   icon: d.data.icon
               }
           })
            return cloudServiceList;
        };
```

reference store를 사용하지 않고 위처럼 변경되어야 할 것 같아요. 
보니까 제가 autocomplete resource api 에 icon을 넣지 않아서 이렇게 하신 것 같아요.
추가해두겠습니다!

Reply:
감사합니다!!!